src/components/pages/order/OrderPage.js

```diff
@@ -10,24 +10,24 @@ import GlobalContext from "../../../context/GlobalContext";
 export default function OrderPage() {
   const { name } = useParams();
   const [isModeAdmin, setIsModeAdmin] = useState(false);
-  const [panelTabIndex, setPanelTabIndex] = useState(2);
-  const [isPanelReduce, setIsPanelReduce] = useState(false);
+  const [panelTabIndex, setPanelTabIndex] = useState(1);
+  const [isPannelCollapsed, setIsPannelCollapsed] = useState(false);
 
-  const contextValue = {
+  const globalContextValue = {
     isModeAdmin,
-    updateIsModeAdmin: setIsModeAdmin,
+    setIsModeAdmin: setIsModeAdmin,
     panelTabIndex,
     setPanelTabIndex,
-    isPanelReduce,
-    setIsPanelReduce,
+    isPannelCollapsed,
+    setIsPannelCollapsed,
   };
 
   return (
-    <GlobalContext.Provider value={contextValue}>
+    <GlobalContext.Provider value={globalContextValue}>
       <OrderPageStyled>
         <NavBar userName={name} />
         <main className="main-container">
-          {/* TODO: basket container */}
+          {/* @TODO: basket container */}
```

Comment:
```suggestion
          {/* Keep this basket container for the basket feature ticket */}
```

ici, ce n'est pas un @TODO à proprement parler car il n'y a aucune action requise. C'est un simple commentaire pour signaler aux autres dev qu'il ne faut pas te rappeler dans la PR de le supprimer.